servers/mu/src/domain/clients/cron.js

```diff
@@ -58,6 +58,9 @@ function initCronProcsWith ({ PROC_FILE_PATH, startMonitoredProcess }) {
 
 function startMonitoredProcessWith ({ logger, CRON_CURSOR_DIR, CU_URL, fetchCron, crank, PROC_FILE_PATH }) {
   return async ({ processId }) => {
+    if (cronsRunning[processId]) {
+      throw new Error('Process already being monitored')
+    }
```

Comment:
How does this bubble up to client? Does the client receive a `4xx` status code? Also, is there an accompanying test for this? @VinceJuliano what ought to be the expected behavior here?

Perhaps the client should receive a `208`? Or perhaps more pessimistically a `409`?